firebase.go

```diff
@@ -65,6 +68,20 @@ func sanitizeURL(url string) string {
 	return url
 }
 
+func sanitizePath(p string) string {
+	// remove slashes from the front and back
+	//	/foo/.json -> foo/.json
+	s := strings.Trim(p, "/")
```

Comment:
Looking through the code, I wasn't debating the use of this function. Don't think it's really used anywhere that couldn't be replaced with `strings.Trim(s, "/")`